definition_of_done.md

```diff
@@ -0,0 +1,38 @@
+ Definition of done
+
+This document clarifies, when an product backlog item is ready. During scrum development process this gives us a checklist,
+what needs to be done.
+The individual points are a basis for acceptance criteria and are shortened or appended by specific measures as needed.
+
+## Documentation
```

Comment:
Where do we document our overall hardware requirements which might undergo changes?